store/store_test.go

```diff
@@ -414,6 +417,12 @@ func TestSaveTxInfo(t *testing.T) {
 			require.Equal(t, block.Height, txInfo.Height)
 			require.Equal(t, uint32(i), txInfo.Index)
 			require.Equal(t, txResponseCodes[i], txInfo.Code)
+			// We don't save the logs for successful transactions
+			if txResponseCodes[i] == 0 {
```

Comment:
[optional] 

```suggestion
			if txResponseCodes[i] == abci.CodeTypeOK {
```

Reply:
resolved in a previous commit! thanks rootul!

Reply:
I don't think this instance was resolved in a previous commit though